tests/unit/Model/Command/UserRecommendationTest.php

```diff
@@ -22,7 +23,7 @@ public function shouldBeInstantiableViaNamedConstructorWithDefaultValues(): void
                     'rotation_rate' => 1.0,
                     'rotation_time' => 3600,
                     'hard_rotation' => false,
-                    'min_relevance' => UserRecommendation::MINIMAL_RELEVANCE_LOW,
+                    'min_relevance' => MinimalRelevance::LOW,
```

Comment:
Typo, should be `LOW()`.

Same applies to `HIGH()` below.

Reply:
No, it shouldn't - the methods returns actual instance of the Enum, while this is just the constant, which contains the value ("low"). This is the same as `MinimalRelevance::LOW()->getValue()`.